go/pkg/clickhouse/README.md

```
@@ -0,0 +1,13 @@
+# Changing schemas
+
+To change a schema, just edit a table in the schema folder.
+Afterwards generate a migration file using atlas.
+Make sure docker is running, so atlas can use it to generate migrations.
+
+```bash
+cd go/pkg/clickhouse
+atlas migrate diff \
+	--dir "file://migrations" \
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)